helm/cluster-openstack/templates/_openstack_cluster_template.tpl

```diff
@@ -10,6 +10,9 @@ spec:
   template:
     spec:
       cloudName: {{ .Values.cloudName }}
+      {{- if .Values.controlPlane.availabilityZones }}
+      controlPlaneAvailabilityZones: {{ join "," .Values.controlPlane.availabilityZones }}
```

Comment:
This is weird because the label for machines is called `failuredomains` - seems to me like upstream uses different terminology for the same feature?

Reply:
Some good info about this situation here https://github.com/kubernetes-sigs/cluster-api-provider-openstack/issues/525

Reply:
I think you might have the wrong type here. This is a slice of strings https://github.com/kubernetes-sigs/cluster-api-provider-openstack/blob/eb288d2a970df348bec5412349638b1869d89069/api/v1beta1/openstackcluster_types.go#L130 but you're giving a single string with values separated by a comma if I'm reading this correctly.

Reply:
Not sure what happened here - for some reason I was convinced on Friday that it was a single string :(